hack/arktos-cni.rc

```diff
@@ -67,11 +67,13 @@ install_cni_bridge() {
     echo "done with bridge cni plugin installation"
 }
 
-cleanup_cni_mizar() {
+cleanup_cni() {
     sudo rm -rf "${cni_bin_dir}"
     [ ! -d "${cni_conf_dir}" ] && sudo mkdir -p "${cni_conf_dir}"
     [ ! -d "${cni_bin_dir}" ] && sudo mkdir -p "${cni_bin_dir}"
     sudo find ${cni_conf_dir}/ -type f -not -name '10-mizarcni.conf' -delete
+    sudo find ${cni_conf_dir}/ -type f -not -name '10-flannel.conflist' -delete
```

Comment:
Can this be completed by remove all files in the directory? If so, we don't need to keep updating the file list when adding new cni plugin. Plus, delete by exact filename could be error prone.

Reply:
Because the permission of two directory /etc/cni and /etc/cni/net.d is "drwx------", which means only root can read/write/executable, all files in the directory /etc/cni/net.d can not deleted in one time whereas all files in the directory /opt/cni/bin can be removed in one time because the permission of /opt/cni/ is "drwxr-xr-x" and permission of /opt/cni/bin is "drwxrwxr-x".

Let me try to change the permission of /etc/cni into 'drwxr-xr-x' and the permission of /etc/cni/net.d into 'drwxrwxr-x', respectively first, then attempt to remove all files in the directory /etc/cni/net.d, then change the permissions of two directories back to original ones.

Reply:
New codes do work so far.

Reply:
New codes do work for all tests.